formulas/functions/statistical.js

```diff
@@ -245,6 +246,41 @@ const StatisticalFunctions = {
 
   MINIFS: () => {},
 
+  MODE: (...numbers) =>{
+    let arr = []
+    arr.push(...numbers.map(a => a.value))
+    const flatArr = FormulaHelpers.flattenDeep(arr),
+          filteredArr = flatArr.filter(a => typeof a === "number");
+
+    if(filteredArr.length <= 0)
+      throw FormulaError.NA;
+
+    let map = new Map();
+    for(let i = 0; i < filteredArr.length; i++){
+      const curr = filteredArr[i];
+      if(map.has(curr))
+        map.set(curr, map.get(curr) + 1);
+      else
+        map.set(curr, 1);
+    }
+
+    const keys = map.keys();
+    let currBig = 0,
+        rv = null,
+        all1 = true;
```

Comment:
```
all1 = keys.all(k => map.get(key) <= 1)
return keys.map(k => [map.get(key), key]).max()[1];
```

Reply:
I dont think I can do this b/c keys is an iterator rather than an array

Reply:
Then convert keys into an array.